docker/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN if grep -q '^ID=alpine$' /etc/os-release; then \
   apk add --no-cache --virtual temp-packages shadow=4.13-r0; \
   fi
 RUN if [ "$(id ${MAMBA_USER} -u)" != "$NEW_MAMBA_USER_ID" ]; then \
-  usermod "-u ${NEW_MAMBA_USER_ID}" "${MAMBA_USER}"; \
+  usermod -o "-u ${NEW_MAMBA_USER_ID}" "${MAMBA_USER}"; \
```

Comment:
@Mehak261124 could you either explain or remove the changes in the docker configuration?